components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -2049,6 +2078,9 @@ bool AdsServiceImpl::StopNotificationTimeoutTimer(
 }
 
 bool AdsServiceImpl::ShouldShowNotifications() {
+  #ifdef OS_WIN
+    return true;
+  #endif
```

Comment:
We should still call `ShouldShowNotifications` in the helper as business logic such as full screen should still be determined on a per platform basis